pkg/parse/listener_impl.go

```diff
@@ -1009,10 +1250,17 @@ func (s *TreeShapeListener) ExitHttp_path(*parser.Http_pathContext) {
 
 // EnterRet_stmt is called when production ret_stmt is entered.
 func (s *TreeShapeListener) EnterRet_stmt(ctx *parser.Ret_stmtContext) {
+	payload := strings.Trim(ctx.TEXT().GetText(), " ")
+	if s.lintMode && !regexp.MustCompile(`<:`).MatchString(payload) {
```

Comment:
this regex will not allow `return ok` or `return error`

Reply:
yep that's the idea. It only warns users so it is still allowed, there's just going to be a linter warning. https://github.com/anz-bank/sysl/issues/882#issue-636695948

Reply:
@anz-gordonj7 I just realised what you meant 😅, I'll try to change it.

Reply:
fixed this